src/components/account/AccountBookDailyItem.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react';
 import styled from '@emotion/styled';
 import { theme } from '@styles';
-import type { SingleAccount } from '@pages/accountBookDaily';
-interface AccountBookDailyItemProps {
-  item: SingleAccount;
-}
-
+import { dateFormatter } from '@utils/formatter/dateFormatter';
+import { useNavigate } from 'react-router-dom';
+import { currencyFormatter } from '@utils/formatter';
 const colorType = {
   INCOME: theme.$blue,
   EXPENDITURE: theme.$red,
 };
 
-const AccountBookDailyItem: React.FC<AccountBookDailyItemProps> = (props) => {
+const AccountBookDailyItem: React.FC<{ item: SingleAccount }> = (props) => {
+  const navigate = useNavigate();
+
   return (
-    <Container>
+    <Container onClick={() => navigate(`/account/update/${props.item.id}`)}>
```

Comment:
혹시 이부분 `/account/update/income/${id}` or `/account/update/expenditure/${id}`로 바꿔주실 수 있으신가요?? id만 받아서는 어떤 타입의 api(수입 수정인지 지출 수정인지)를 호출해야 하는지 알 수가 없더라고요! 제가 router쪽은 작업중인 브랜치에서 바꿔놔가지고 라우팅 되는 부분만 바꿔주시면 될 것 같아요

Reply:
아하 수정 페이지가 어떤식으로 진행되는 지 몰라서 임의로 작업해놨어요!
`income` | `expenditure` 두 타입이 맞나요?
```suggestion
    <Container onClick={() => navigate(`/account/${props.item.type}/${props.item.id}`)}>
```
이렇게 수정해둘게요!

Reply:
소문자로 해주세요! 제가 대문자로 한번 넣어봤었는데 인식 못하는 것 같아요 ㅎㅎ

Reply:
수정해두었습니다~! 0c91c3180e58cbc6618799902ac7de1192accbc1